src/environments/atari.jl

```diff
@@ -148,10 +132,9 @@ function RLBase.reset!(env::AtariEnv)
 end
 
 
-imshowgrey(x::Array{UInt8,2}) = imshowgrey(x[:], size(x))
+imshowgrey(x::Array{UInt8,2}) = imshowgrey(reshape(x, :), size(x))
```

Comment:
This will turn x into a view and I guess it will trigger a method not found error.

Reply:
Ah, yes, you're right...